docs/table_design/Data_distribution.md

```diff
@@ -112,51 +214,208 @@ CREATE TABLE site_access(
 
-- The column with low cardinality.
-- The column that is often used as a filter in queries.
-- The amount of data in each partition must be less than 100 GB.
+```SQL
+RECOVER PARTITION p1 FROM site_access;
+```
 
-### Bucketing
+#### View partitions
 
-Data in partitions can be subdivided into multiple buckets, and the data within a bucket can be referred to as a tablet.
+The following statement returns details of all partitions in table `site_access`.
 
-- Bucketing methods: supports Random bucketing (since v3.1) and Hash bucketing.
-Random bucketing: does not require the bucketing key when creating a table or adding a partition. The data is randomly distributed across different buckets.
-- Hash bucketing: requires the bucketing key when creating a table or adding a partition. Data within a partition is subdivided into buckets based on the bucketing key. Rows with the same bucketing key value are assigned to the corresponding and unique bucket.
+```SQL
+SHOW PARTITIONS FROM site_access;
+```
 
-The number of buckets: By default, StarRocks automatically sets the number of buckets (since version 2.5.7). However, you can also manually set the number of buckets. For more information, see [Determine the number of buckets](#determine-the-number-of-buckets).
+## Configure bucketing
 
-#### Random bucketing (since v3.1)
+### Random bucketing (since v3.1)
 
-For data in a partition, StarRocks distributes the data randomly across all buckets, which is not based on specific column values. This method is suitable for scenarios with small data volumes and low requirements for query performance. If you do not set a bucketing method, StarRocks uses random bucketing by default and automatically determines the number of buckets.
+StarRocks distributes the data in a partition randomly across all buckets. It is suitable for scenarios with small data sizes and relatively low requirement for query performance. If you do not set a bucketing method, StarRocks uses random bucketing by default and automatically sets the number of buckets.
 
-However, note that the query performance provided by random bucketing may not be ideal when you query massive amounts of data and frequently use certain columns as conditional columns. In this scenario, it is recommended to use [hash bucketing](#hash-bucketing). Because only a small number of buckets need to be scanned and computed, significantly improving query performance.
+However, note that if you querymassive amounts of data and frequently use certain columns in the filtering condition, the query performance provided by random bucketing may not be optimal. In such scenarios, it is recommended to use [hash bucketing](#hash-bucketing). When querying with these columns in the filtering condition, only data in a small number of buckets that the query hits need to be scanned and computed, which can significantly improve query performance.
 
-##### Precautions
+**Precautions**
```

Comment:
![image](https://github.com/StarRocks/starrocks/assets/57167462/bdf7a221-9a75-408d-938f-abe016cf743e)
I think the following are "Limits" rather than "Precautions".